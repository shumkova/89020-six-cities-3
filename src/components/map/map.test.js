import React from "react";
import renderer from "react-test-renderer";
import Map from "./map";

const coordinates = [
  [52.3909553943508, 4.85309666406198],
  [52.369553943508, 4.85309666406198],
  [52.3909553943508, 4.929309666406198],
  [52.3809553943508, 4.939309666406198],
];

const cityCords = [52.38333, 4.9];

it(`Map should render correctly`, () => {
  const tree = renderer.create(
      <Map
        cityCords={cityCords}
        coordinates={coordinates}
      />, {
        createNodeMock: () => {
          return document.createElement(`div`);
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
