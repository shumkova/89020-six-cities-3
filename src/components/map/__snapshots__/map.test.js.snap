// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map should render correctly 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
